installer_windows.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# make protobufs for logging part first
+make all
+
+# get tensorflow
+git clone https://github.com/tensorflow/tensorflow
+cd tensorflow
+
+# run configuration.
+bash configure --prefix=/mingw
+
+# build tensorboard
+#bazel build tensorflow/tensorboard:tensorboard
+bazel build -c opt --cpu=x64_windows_msvc --host_cpu=x64_windows_msvc --copt="/w" --verbose_failures tensorflow/tensorboard:tensorboard
+
+# prepare pip installation package
+cp -r ../tools/* bazel-bin/tensorflow/tools/
+
+# get .whl file in python/dist/
+bash bazel-bin/tensorflow/tools/pip_package/build_pip_package.sh ../python/dist/

```

Comment:
In `build_pip_package.sh`, `.runfiles` in Windows are placed in a different location. But I forgot to handle this in `tensorboard-binary.patch`, could you take a look? Just take a look at the `tensorboard` binary and try to handle runfiles location in `FindModuleSpace`.